recipes/cartopy_offlinedata/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "cartopy_offlinedata" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/willirath/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: e7f3a6d47d97e133c516c631c1dbb366300c43f0dcae0d5c49c8a2a5810c477e
+
+build:
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+    - cartopy
```

Comment:
Should have a bash dependency

Reply:
I can add it but isn't it going to use `/bin/bash` anyway?

Reply:
Not if you change it to use `/usr/bin/env bash` :wink:

Reply:
Fair point.  I changed several shebangs to this format.  Not sure it matters in the big picture but it's a better practice.

Reply:
Should be resolved.  I won't change the shebang upstream in `download_cartopy_data.sh` but this could be part of a future update.

Reply:
Ahh yeah, then bash needs to be in the run deps

Reply:
Like I said in my notes below, I don't expect any users to want to run that script, it's just installed as part of the package for situations where the data it downloads isn't already shipped with the package.  But I added bash to the run dependencies as you ask.  Anything else?